cmd/promtool/main.go

```diff
@@ -303,7 +328,7 @@ func main() {
 		os.Exit(backfillOpenMetrics(*importFilePath, *importDBPath, *importHumanReadable, *importQuiet, *maxBlockDuration))
 
 	case importRulesCmd.FullCommand():
-		os.Exit(checkErr(importRules(*importRulesURL, *importRulesStart, *importRulesEnd, *importRulesOutputDir, *importRulesEvalInterval, *maxBlockDuration, *importRulesFiles...)))
+		os.Exit(checkErr(importRules(serverURL, *importRulesStart, *importRulesEnd, *importRulesOutputDir, *importRulesEvalInterval, *maxBlockDuration, *importRulesFiles...)))
```

Comment:
Can this one also take the roundtripper?

Reply:
:flushed: sorry about that, not sure how that slipped thought… 

Thanks for noticing, should be fixed  now

Reply:
https://github.com/prometheus/prometheus/pull/11487/commits/f1de2cec3d34a94d594ae727e86cd18ef44d8f39